cns/middlewares/k8sSwiftV2.go

```diff
@@ -156,85 +155,3 @@ func (k *K8sSWIFTv2Middleware) validateIPConfigsRequest(ctx context.Context, req
 	// retrieve podinfo from orchestrator context
 	return podInfo, types.Success, ""
 }
-
-// getIPConfig returns the pod's SWIFT V2 IP configuration.
-func (k *K8sSWIFTv2Middleware) getIPConfig(ctx context.Context, podInfo cns.PodInfo) ([]cns.PodIpInfo, error) {
```

Comment:
Why is this section being removed from here? This does not change for windows or linux.

Reply:
implementing this function in k8sswiftv2_linux & _windows, but will change that now & only move a small section of this function into other 2 files